test/fixtures/check-direct-import-transitive-dependency.txt

```diff
@@ -1,8 +1,8 @@
 purs compile: No files found using pattern: test/**/*.purs
 [34m[info] [0mBuild succeeded.[0m
-[33m[warn] [0mNone of your project files import modules from some projects that are in the direct dependencies of your project.
-These dependencies are unused. To fix this warning, remove the following packages from the list of dependencies in your config:
-- effect[0m
```

Comment:
This is covered in the following `Spago should warn on unused dependencies` test.
Could add this back in if this is something we want to test, or perhaps make another test for both unused and missing dependencies.

Reply:
Yep I'd like to keep this warning here too, I'd say we could add `effect` back